src/main/java/martin/Ui.java

```diff
@@ -0,0 +1,37 @@
+package martin;
+import java.util.Scanner;
+
+public class Ui {
+
+    private Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void showWelcome() {
+        printMessage("Hello! I'm Martin\n     What can I do for you?");
+    }
+
+    public String readCommand() {
+        return scanner.nextLine().trim();
+    }
+
+    public void showLine() {
+        System.out.println("    ____________________________________________________________");
```

Comment:
Perhaps adding the Line string as a constant would make your code more readable and reusable